dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1000,7 +999,6 @@ data:
             },
             {
               "datasource": {
-                "name": "Expression",
```

Comment:
Is this removal expected?

Reply:
I went back over all the changes in the commit before last and changed some smaller items that shouldn't have been changed with the new configmap.